Makefile

```diff
@@ -34,6 +34,9 @@ build: manifests generate kubectl-gadget gadget-container
 .PHONY: phony_explicit
 phony_explicit:
 
+assert-no-uncommitted-changes:
+	@if [[ ! -z "$(changes)" ]] ; then git status --porcelain ; false ; fi
```

Comment:
Do you think people would be really using `make assert-no-uncommitted-changes`? What about putting this command directly on the github actions yml file?

Reply:
I can move it to the github actions yaml file.

My thinking was that the logic to detect changes is in the Makefile, so the logic gets changed it's better to have the check implemented in the same file to help avoid discrepancies.